libnmstate/metadata.py

```diff
@@ -27,6 +27,8 @@
 MASTER = '_master'
 MASTER_TYPE = '_master_type'
 ROUTES = '_routes'
+DNS_METADATA = '_dns'
+DNS_PRIORITY = '_priority'
```

Comment:
I guess the data is embedded under `DNS_METADATA`, but it is hard to understand it from here. Perhaps name it `DNS_METADATA_PRIORITY`.